src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/mgmt_vm/nested_templates/nsg_existing_or_none.json

```diff
@@ -2,6 +2,9 @@
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
+    "location": {
+      "type": "string"
+    },
```

Comment:
I thought about factoring the NSG out, but the rules we auto-add only make sense for in this case.  Adding SSH/RDP for a generic NSG is less compelling, so I left it seperate

---
In reply to: [73382783](https://github.com/Azure/azure-cli/pull/599#discussion_r73382783) [](ancestors = 73382783)